README.md

```diff
@@ -185,8 +185,10 @@ This strategy replaces the `inserts_only` setting in previous versions of dbt-cl
 As a result duplicate rows are not eliminated, and there is no temporary or intermediate table.  It is the fastest approach if duplicates are either permitted
 in the data or excluded by the incremental query WHERE clause/filter.
 
-### The insert_overwrite Strategy
-
+### The insert_overwrite Strategy (Experimental)
+> [IMPORTANT]  
+> The insert_overwrite strategy wasn't tested and verified with a cluster setup. 
```

Comment:
`wasn't` -> `was not` 

We should clarify whether it works and was tested in ClickHouse Cloud